### PR TITLE
translation: Update chapter_data_structure/classification_of_data_structure.md

### PR DESCRIPTION
**Pull request by yanedie**
If this PR is related to coding or code translation, please fill out the checklist and paste the console outputs to the PR.

- [ ] I've tested the code and ensured the outputs are the same as the outputs of reference code.
- [ ] I've checked the code (formatting, comments, indentation, file header, etc) carefully.
- [ ] The code does not rely on a particular environment or IDE and can be executed on a standard system (Win, macOS, Ubuntu).

### CODE REVIEW

File: docs-en/chapter_data_structure/classification_of_data_structure.md [hunk 1]
```diff
@@ -1,46 +1,48 @@
...
-- **Tree structure**: tree, heap, hash table with one-to-many relationships between elements.
-- **Graph**: graph with many-to-many relationships between elements.
+- **Linear structures**: arrays, linked lists, queues, stacks, hash tables, with one-to-one sequential relationship between elements.
+- **Tree structure**: tree, heap, hash table, with one-to-many relationship between elements.
+- **Graph**: graph with many-to-many relationship between elements.
 
 ## Physical Structure: Continuous vs. Dispersed
 
-In computers, memory and hard disk are the two main storage hardware devices. Hard disks are used primarily for long-term data storage, and are larger (usually up to the terabyte degree) and slower. Memory is used for temporary storage of data while running a program, and is faster, but has a smaller capacity (usually at the GB level).
+**When an algorithm is running, the data being processed is stored in memory**. The figure below shows a computer memory module where each black square represents a memory space. We can think of the memory as a giant Excel sheet in which each cell can store data of a certain size.
```
Comment (K3v123) [inline docs-en/chapter_data_structure/classification_of_data_structure.md:24]: Hi yanedie
the word "below" can be changed to "3-2" to make it more specific, can maintains the consistency of the CN version.

**Review by K3v123 — COMMENTED** [review]
overall I think its really good. what i suggested are just optional and some minor suggestions. its consistent to the CN version.
But I'm not an expert at this, so might need to be rechecked.

File: docs-en/chapter_data_structure/classification_of_data_structure.md (hunk 1 above)
Comment (krahets) [reply docs-en/chapter_data_structure/classification_of_data_structure.md:24]: Hi @K3v123, the figures will be numbered automatically in the build script. We don't need to manually modify it.

**Review by krahets — APPROVED** [review]
Thanks for the PR and the review by @K3v123 !

File: docs-en/chapter_data_structure/classification_of_data_structure.md (hunk 1 above)
Comment (K3v123) [reply docs-en/chapter_data_structure/classification_of_data_structure.md:24]: uh... I see, haha. thanks for letting me know Krahets. (；・∀・)b

**Pull request merged**